llama_index/query_engine/retriever_query_engine.py

```diff
@@ -67,6 +68,7 @@ def from_args(
         output_cls: Optional[BaseModel] = None,
         use_async: bool = False,
         streaming: bool = False,
+        output_parser: Optional[BaseOutputParser] = None,
```

Comment:
@zsimjee actually, output parsers should be set on the `llm_predictor` of the service context, by using a `StructuredLLMPredictor`

Although also been meaning to unify how those abstractions work 😅